tests/smoke/README.md

```diff
@@ -0,0 +1,15 @@
+# Addons-Frontend Smoke Tests
+
+A simple smoke test suite for the addons-frontend docker image.
+
+### Prerequisites
+
+Python 3 is recommended but the tests can run on Python 2. You must have `tox` installed either way.
+
+You should also stop any services that may be using the port `3000`. You will also need to add `olympia.test` to your `/etc/hosts` in line with `127.0.0.1`.
```

Comment:
Hmm, I guess port 3000 isn't an issue anymore? It looks like the server is exposed on port 80.

Reply:
It is still needed as that is how I can get all the images to work together locally.